src/spikeinterface/sorters/runsorter.py

```diff
@@ -545,6 +553,7 @@ def run_sorter_container(
     cmd = ["python", f"{in_container_script_path_unix}"]
     res_output = container_client.run_command(cmd)
     run_sorter_output = res_output
+    run_sorter_output = run_sorter_output[2:-1].replace("\\n", "\n").replace("\\'", "'")
```

Comment:
Why this ?
Is it system specific or terminal specific ?
Could you add a comment to remember ?

Reply:
done!